packages/paste-core/components/button/src/utils.tsx

```diff
@@ -0,0 +1,13 @@
+const PROPS_TO_BLOCK = ['className', 'style', 'height', 'width', 'loading'];
```

Comment:
As this is such a small list of props (of which I'm not sure loading should be included) I'm tempted to just set these properties to null on the button itself rather than filter them out.

Reply:
I like this idea better. I added `loading` here because when I added the safely spread function I was getting the following console error:

![image](https://user-images.githubusercontent.com/1350081/80387901-7178b800-8866-11ea-8ebb-9d7cd5706d0b.png)

It may have something to do with the loading html attribute (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes):
![image](https://user-images.githubusercontent.com/1350081/80388012-979e5800-8866-11ea-9ee7-98a1dd86bc3b.png)

Reply:
This might just be something to do with the way the storybook Boolean knob is working

Reply:
Similar error when running tests:
![image](https://user-images.githubusercontent.com/1350081/80396858-8a876600-8872-11ea-95a3-a427d0f51bbd.png)

Reply:
OK, so we don't actually use `loading` spread to the button element. You're going to have to rework the way the props come in and selective destructure some off of props before spreading.

You'll need to pull loading and disabled off of props for example, do to the way we handle them as states, not attributes on the button element. See getButtonStates(props) on line 46ish